Source/MonoGame.Extended.Tests/AngleTest.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using Microsoft.Xna.Framework;
+using NUnit.Framework;
+
+namespace MonoGame.Extended.Tests
+{
+    [TestFixture]
+    public class AngleTest
+    {
+        private const float DELTA = 0.0000001f;
```

Comment:
> Maybe we could push the .DotSettings files of the various projects as well.
Very good idea, it would help a lot for people like me who are used to different conventions and schemes. I don't know how that works exactly but it's definitely worth looking into